spec.md

```diff
@@ -197,13 +197,11 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
                "unit":"1GB of messages over 20GB"
             }
          ],
-        "bullets": [{
-            "content": "Shared fake server"
-          }, {
-            "content": "5 TB storage"
-          }, {
-            "content": "40 concurrent connections"
-        }]
+        "bullets": [
```

Comment:
I tried to explain in the PR description, but, while this is a blob and yes, this is technically valid, it is:
 1. silly the way it's written
 2. does not comform to the existing Cloud Foundry convention, where bullets is an array of strings, not a map, so unless there's a strong reason for it to diverge, seemed like they should at least be consistent.
https://docs.cloudfoundry.org/services/catalog-metadata.html

Their examples are:
              "metadata":{
                  "bullets":[
                     "20 GB of messages",
                     "20 connections"
                  ],

Which makes more sense than duplicating 'content' over and over for no reason. It's confusing.